### ISSUE TITLE
[Bug Bash] In half screen mode button content is obscured

### ISSUE THREAD

**Issue by protikbiswas100** [microsoft/react-native-gallery#726]
Steps to reproduce:

1. Open the app 
2. Snap window to half screen 

Actual Result:
Using the app in half-screen mode, none of the button contents are available. 
 
Again ac A11y issue as the content is not visible and in lower resolutions it will cause an issue. 

Expected Result:
Buttons content should not be obscured when enough space is available.

**Comment by protikbiswas100** [comment]
https://github.com/microsoft/react-native-gallery/pull/738/
